forch/device_report_client.py

```diff
@@ -50,30 +55,48 @@ def __init__(self, result_handler, target, unauth_vlan, tunnel_ip):
 
     def start(self):
         """Start the client handler"""
-        grpc.channel_ready_future(self._channel).result(timeout=CONNECT_TIMEOUT_SEC)
-        self._stub = SessionServerStub(self._channel)
+        # Context may be set already
```

Comment:
I'm not sure what to make of this comment. Can you expand it a bit or position it better?  Is this to say that if the context is already set then this will throw a RuntimeError?  If so, I'd either clarify that in the comment or move to the except clause.